test/exprtest/blas1_axpy_copy_test.cpp

```diff
@@ -23,6 +23,7 @@
  *
  **************************************************************************/
 #include "blas_test.hpp"
+#include "blas_test_macros.hpp"
```

Comment:
The other tests don't include this file as it's already included by `blas_test.hpp`. Can we remove them?

Reply:
My bad, removed it.